types.go

```diff
@@ -0,0 +1,63 @@
+package fq
+
+import "time"
+
+// TODO(aaron-prindle) currently testing with one concurrent request
+const C = 1 // const C = 300
+
+const G = 100000 //   100000 nanoseconds = .1 milliseconds || const G = 60000000000 nanoseconds = 1 minute
+
+// Packet is a temporary container for "requests" with additional tracking fields
+// required for the functionality FQScheduler as well as testing
+type Packet struct {
+	item      interface{}
+	size      int
```

Comment:
Why is this an `int`?  What units is it in?

I see this field being used only in tests, where all the calculations are ultimately done in `float64`; why not start with `float64` values?

Reply:
Currently this isn't used.  I originally adaped these tests from a wfq implementation that varied the size of packets in this test.  This 'size int' was the # of bytes.  I was considering adapting this varying byte size to be varying request time for our implementation but have not done so yet.  I have changed this value to be float64 and have changed the name from size to servicetime but it is still unused currently.